applications/ws-affiliation-rnsr/Dockerfile

```
@@ -1,29 +1,23 @@
 # syntax=docker/dockerfile:1.2
 FROM inistcnrs/lodex-workers-python:4.0.11 as build1
-RUN --mount=type=secret,id=webdav_login \
-    --mount=type=secret,id=webdav_password \
-    --mount=type=secret,id=webdav_url \
+RUN --mount=type=secret,id=webdav_login cat /secrets/login \
+    --mount=type=secret,id=webdav_password cat/secrets/password \
+    --mount=type=secret,id=webdav_url cat/secrets/url \
     pip install dvc[webdav]==2.48 && \
-    source logins.env && \
-    
-    #REPRENDRE ICI
-
-
-
-
+    git init && \
+    dvc init && \
     dvc remote add -d webdav-remote "${WEBDAV_URL}" && \
```

```suggestion
    dvc remote add -d webdav-remote $(cat /run/secrets/webdav_url) && \
```

Le mécanisme de secrets de buildkit, apparemment quelle que soit la source du secret (`src` ou `env`) stocke les valeurs dans des fichiers du répertoire `/run/secrets` dont le nom est l'identifiant du secret.

`cat /run/secrets/secret_id` permet donc de mettre sur la sortie standard la valeur qu'on y a mise.
Et la syntaxe `$(...)` est au final remplacée par la sortie de la commande qui est à l'intérieur.